Makefile

```diff
@@ -25,6 +25,9 @@ format: install
 test: install
 	poetry run nox -s test $(if $(PYTHON),--python=$(PYTHON),)
 
+benchmark: install
```

Comment:
That's really cool we actually have our first benchmark test in this repo

But do you think it deserves to be in `Makefile`?

Reply:
yes, why not?